lib/pbench/agent/tool_meister_client.py

```diff
@@ -124,7 +129,9 @@ def __exit__(self, *args):
         """On context exit, just close down the to SignalExporter object."""
         self.sig_pub.shutdown()
 
-    def publish(self, group: str, directory: str, action: str, args=None) -> int:
+    def publish(
+        self, group: str, directory: Union[str, Path, None], action: str, args=None
+    ) -> int:
```

Comment:
The last parameter is missing its type hint...something like:
```
    def publish(
        self,
        group: str,
        directory: Union[str, Path, None],
        action: str,
        args: Any = None,
    ) -> int:
```
(Sadly, Black completely wraps it....)